armi/physics/neutronics/parameters.py

```diff
@@ -660,14 +684,16 @@ def linPowByPinGamma(self, value):
             "detailedDpaPeakRate",
             units="DPA/s",
             description="Peak DPA rate based on detailedDpaPeak",
-            location=ParamLocation.AVERAGE,
+            location=ParamLocation.MAX,
+            categories=["detailedAxialExpansion", "neutronics"],
```

Comment:
do you want the string version here or `parameters.Category.detailedAxialExpansion`, etc?

Reply:
Yeah we should use `parameters.Category.detailedAxialExpansion` for anything new that's going in. I won't go through and change pre-existing string versions though.